src/canvas/canvas.controller.ts

```diff
@@ -270,4 +271,34 @@ export class CanvasController {
     const userId = dto.userId;
     await this.canvasService.cancelAccess({ canvasId, userId });
   }
+
+  /**
+   * Adds a single tag to a canvas
+   * @param canvasId
+   * @param dto - an object containing 'tagId'
+   */
+  @Post('/:id/tags')
+  @UseGuards(AuthenticatedGuard, CanvasGuard)
+  public async addTag(
```

Comment:
I feel like we should have a way to assign multiple tags to given canvas in a single request. This would be quite useful on the front-ends.

Maybe we can change this endpoint to accept a list of tags?

Reply:
Done